bin/utils/validations.dart

```diff
@@ -0,0 +1,8 @@
+
+/// Validations rules
+class Validations {
+  // function that returns true always
+  static bool none(dynamic s){
+    return true;
+  }
+}
```

Comment:
Why no validations for input ? 
If no validations then this class isn't required